lib/shared/relative-module-resolver.js

```diff
@@ -3,42 +3,41 @@
  * @author Teddy Katz
  */
 
-"use strict";
-
-const Module = require("module");
+import Module from "module";
 
 /*
  * `Module.createRequire` is added in v12.2.0. It supports URL as well.
  * We only support the case where the argument is a filepath, not a URL.
  */
-// eslint-disable-next-line node/no-unsupported-features/node-builtins, node/no-deprecated-api
+// eslint-disable-next-line node/no-deprecated-api
 const createRequire = Module.createRequire || Module.createRequireFromPath;
```

Comment:
`Module.createRequireFromPath` is no longer needed, as node <12.2.0 will be dropped?

Reply:
Sure. I don't think I was sure at the time, so belive I just went with parity with another project using it, but sure, I think makes sense to remove. Removed...